data/tekton-pipelines/okd/windows11-installer-example-pipelinerun.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: win11
+  namespace: kubevirt
+  labels:
+    tekton.dev/pipeline: win11
+spec:
+  status: "PipelineRunPending"
+  params:
+    - name: winImageDownloadURL
+      value: >-
+       <INSERT_WIN_11_ISO_URL>
+    - name: autounattendConfigMapName
+      value: windows11-autounattend
+    - name: virtioContainerDiskName
+      value: >-
+        quay.io/kubevirt/virtio-container-disk@sha256:3e88b9bde8792ba21d6cdaad70c79d5e9b339ef676402e932c6fcc2bda100d04
```

Comment:
You can replace the tag with latest or the kubevirt version. This was needed when the updated virtio-container-disk was not included in a release yet.